tracer/test/Datadog.Trace.Debugger.IntegrationTests/Helpers/DebuggerTestHelper.cs

```diff
@@ -107,11 +107,11 @@ internal static IEnumerable<(ProbeAttributeBase ProbeTestData, LogProbe Probe)>
         return snapshotMethodProbes;
     }
 
-    internal static LogProbe CreateBasicLogProbe(DeterministicGuidGenerator guidGenerator)
+    internal static LogProbe CreateBasicLogProbe(DeterministicGuidGenerator guidGenerator, string overridenProbeId = null)
```

Comment:
Could this be a bit cleaner if we just pass in a `probeId` instead?

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.

Reply:
Done that just now.